content/sensu-go/6.6/release-notes.md

```diff
@@ -99,6 +100,27 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.6.4 release notes
+**January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.3, is now available for download.
+
+Sensu Go 6.6.4 includes a number of bug fixes, security improvements and the addition of the new metric `sensu_go_event_metric_points_processed`. Fixes in this patch release will help prevent the backend from crashing when PostgreSQL is taken offline and will prevent backend entity rows from filling up the entities table. In addition, fixes will further secure the web interface.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.6.4.
+
+**IMPROVEMENTS**
+- ([Commercial feature][259]) In the web UI, added [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe; loaded in this manner, it can be susceptible to tailored click jacking attacks. 
```

Comment:
```suggestion
- ([Commercial feature][259]) In the web UI, added the [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe to prevent tailored click-jacking attacks. 
```